### PR TITLE
renaming master into main

### PR DESCRIPTION
**Pull request by bam241**
this change all mentioned to master (branches or file name) into main

### CODE REVIEW

**Comment by opotowsky** [comment]
Oh, looks like you did this already, @bam! I am trying to build it and not succeeding. Did you build successfully?

**Comment by bam241** [comment]
@opotowsky sorry I forgot about this :(

**Comment by bam241** [comment]
but I still can't change the branch that is used to deploy the website...

**Comment by opotowsky** [comment]
True. Were you able to build locally? I'm getting a failure on it saying `main_doc` is undefined

**Comment by gonuke** [comment]
One of the challenges with this kind of change is that some of them need to be properly imported into the `main` branch in parallel...

File: source/conf.py [hunk 1]
```diff
@@ -39,8 +39,8 @@
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
 
-# The master toctree document.
-master_doc = 'index'
```
Comment (gonuke) [inline source/conf.py:43]: not sure if we can change this - Sphinx may expect this variable to be set (although I'm not sure)

https://www.sphinx-doc.org/en/master/usage/configuration.html

**Review by gonuke — CHANGES_REQUESTED** [review]
I think there is just one thing to confirm and we can merge this.  I've already flipped the bit on the repo to use the `main` branch for publishing

**Comment by gonuke** [comment]
This was effectively accomplished in move to new website.

**Pull request closed without merging**